config/application.yml

```diff
@@ -1,3 +1,6 @@
 test:
   stripe_secret_key: "sk_test_OGO6Th4XlLrPTcNjOSyVOfqx"
   stripe_publishable_key: "pk_test_kNmTDcRkjtxdIdE4fYhjavNw"
+development:
```

Comment:
This `config/application.yml` should be 100% gitignored. Not a file that is supposed to be shared.